app/executor/executor.py

```diff
@@ -0,0 +1,62 @@
+"""Module with executor class."""
+
+import io
+import sys
+import time
+import psutil
+
+
+class Executor:
+    """Class that conects the judge with the code."""
+
+    def __init__(self, output, input_data, code):
+        self.output = output
+        self.input_data = input_data
+        self.code = code
+
+    def judge(self) -> dict:
+        """Method that runs the code on the judge."""
+
+        start_time = time.time()
+
+        memory_before = psutil.virtual_memory().available
+
+        result = self.executioner()
+
+        end_time = time.time()
+
+        memory_after = psutil.virtual_memory().available
+
+        if result:
+            return {
+                "execution_time": end_time - start_time,
+                "memory_used": (memory_before - memory_after) / (1024**2),
+                "verdict": "AC",
+            }
+        return {
+            "execution_time": end_time - start_time,
+            "memory_used": memory_before - memory_after,
+            "verdict": "WA",
+        }
```

Comment:
```suggestion
return {
    "execution_time": end_time - start_time,
    "memory_used": (memory_before - memory_after) / (1024 ** 2) if result else memory_before - memory_after,
    "verdict": "AC" if result else "WA"
}
```

Puede simplificarse un poco